blinds/blinds_api.py

```diff
@@ -14,20 +14,28 @@
 Author: Alex (Yin) Chen
 Creation Date: February 1, 2020
 '''
+
 from requests import codes as RESP_CODES
 from piserver.api_routes import *
 from enum import Enum
 from blinds.blinds_schedule import BlindsSchedule
+import controlalgorithm.angle_step_mapper
```

Comment:
I think you can remove this line, you import the AngleStepMotor right below it directly.

Reply:
I included it for the global const ANGLE_POSITION_FACTOR in angle_step_mapper.
Alex wanted it as a const, but I'm not sure of the best way to do this